api/utils.go

```diff
@@ -23,53 +23,48 @@ func CalculateFileSize(c *gin.Context) {
 	fileHandler, err := c.FormFile("file")
 	if err != nil {
 		FailOnError(c, err)
+		return
 	}
 	size := utils.CalculateFileSizeInGigaBytes(fileHandler.Size)
 	c.JSON(http.StatusOK, gin.H{
-		"file_size_gb":    size,
-		"file_size_bytes": fileHandler.Size,
-	})
-}
-
-// FailNoExist is a failure used when somethign does not exist
-func FailNoExist(c *gin.Context, message string) {
-	c.JSON(http.StatusBadRequest, gin.H{
-		"error": message,
+		"code": http.StatusOK,
+		"response": gin.H{
+			"file_size_gb":    size,
+			"file_size_bytes": fileHandler.Size,
+		},
 	})
 }
 
 // FailNoExistPostForm is a failure used when a post form does not exist
 func FailNoExistPostForm(c *gin.Context, formName string) {
+	err := fmt.Errorf("%s post form not present", formName)
 	c.JSON(http.StatusBadRequest, gin.H{
-		"error": fmt.Sprintf("%s post form not present", formName),
+		"code":     http.StatusBadRequest,
+		"response": err.Error(),
```

Comment:
I dont think this change is needed, isn't this the same as `"response": fmt.Sprintf("%s post form not present", formName)`?